src/configure.py

```diff
@@ -97,8 +100,23 @@ def onservicefee(input):
         global parser
         parser.set(SERVICE_FEE, float(input))
         parser.validate_service_fee(parser.get_conf_obj())
+    except Exception as e:
+        printe(f"Invalid service fee: {str(e)}\nPlease enter a value between 0 and 100.")
+        return
+
+    fsm.go()
+
+
+def onrewardstype(input):
+    if not input:
+        input = 'actual'
+
+    try:
+        global parser
+        rt = RewardsType(input.lower())
+        parser.set(REWARDS_TYPE, str(rt))
     except Exception:
-        printe("Invalid service fee: " + traceback.format_exc())
+        printe("Invalid option for rewards type. Please enter 'actual' or 'ideal'.")
```

Comment:
Considering the fact that there may be more reward type options in the future, Listing enum values would be more future proof. 
Since new reward types are not very likely, not a blocker.

Reply:
I agree and will keep this in mind.